src/GEOPMBenchConfig.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2015 - 2023, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "GEOPMBenchConfig.hpp"
+
+#include <stdlib.h>
+
+namespace geopm
+{
+    const GEOPMBenchConfig &geopmbench_config(void)
+    {
+        static GEOPMBenchConfigImp instance;
+        return instance;
+    }
+
+    GEOPMBenchConfigImp::GEOPMBenchConfigImp()
+        : GEOPMBenchConfigImp(getenv("GEOPMBENCH_NO_MPI") == nullptr)
```

Comment:
This should be included in the geopmbench.1 man page.

Reply:
Just pushed a patch.